mod-orders-storage/examples/routing_list_collection.sample

```diff
@@ -0,0 +1,15 @@
+{
+    "routingLists": [
+      {
+        "id": "c0d13648-347b-4ac9-8c2f-5bc47248b87e",
+        "name": "List name",
+        "notes": "Some note",
+        "userIds": [
+          "d926d900-e27d-46d6-bba8-31e9d5c2cf44",
+          "077274ad-6b4f-4c28-9779-6d381e7a1ca1"
+        ],
+        "poLineId": "8343e5a0-fed8-11e8-8eb2-f2801f1b9fd1"
```

Comment:
We also should ask about supporting of package orders, if this support is needed we also should add titleId in the schema